### ISSUE TITLE
Diff view crashes language server

### ISSUE THREAD

**Issue by ajansveld** [PowerShell/PowerShellEditorServices#544]
The PowerShell language server crashes when using the [Hg extension](https://github.com/mrcrowl/vscode-hg) and viewing a diff. This is similar to this issue: https://github.com/PowerShell/vscode-powershell/issues/729 and probably related to #342.

From the extension's source code it looks like it is sending a uri with a scheme of 'hg'. Can we add this scheme as a short term fix (see e.g. the fix for git https://github.com/PowerShell/PowerShellEditorServices/commit/8f8209534a4e50317af3709bccaadee19a1bf5ba) until #342 is resolved?

Thanks!

**Comment by daviwil** [comment]
Hey ajansveld, I'll look into getting a quick fix done for this for the next update.

**Comment by ajansveld** [comment]
Thanks! I just found that the same issue occurs when using the "Compare with Saved" feature in vscode 1.15, even when the Hg extension is not installed. I will update the title accordingly.

I have attached the logs; unfortunately they do not show the file path that is causing the exception (perhaps it should?).

[vscode-powershell.zip](https://github.com/PowerShell/PowerShellEditorServices/files/1225577/vscode-powershell.zip)
